plugin.go

```diff
@@ -38,30 +47,109 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	var port string
 	var proxyClient string
 	var useSwaggerUI bool
+	var background bool
 	var cmd = &cobra.Command{
-		Use:   "web",
-		Short: "Starts web server",
+		Use:       "web",
+		Short:     "Starts web server",
```

Comment:
also need to test with several instances (ui started on different platforms dir)

probably that requires port auto increment if taken

Reply:
```
➜  ski-platform git:(master) plasmactlui web        

Web running in background with PID: 37089           

➜  zen-opt-platform git:(master) plasmactlui web
                                                
Error: web server port 8080 you are trying to use is not available
                           

```